docs/workflows/ios-specific-workflow-steps/tuist.md

```diff
@@ -20,19 +21,38 @@ Before you run the **Tuist** step, you must complete certain prerequisites, as d
+
+- [**Xcodebuild for Devices**](/workflows/ios-specific-workflow-steps/xcodebuild-for-devices)
+- [**Xcodebuild for iOS Simulator**](/workflows/ios-specific-workflow-steps/xcodebuild-for-ios-simulator)
+- [**Xcodebuild for Testing**](/workflows/ios-specific-workflow-steps/xcodebuild-for-testing)
+- [**Xcodebuild for Unit and UI Testing**](/workflows/ios-specific-workflow-steps/xcodebuild-for-unit-and-ui-test)
+- [**Cocoapods Install**](/workflows/ios-specific-workflow-steps/cocoapods-install)
+
+For more iOS specific workflow steps, please visit the [**iOS Integration**](/workflows/ios-specific-workflow-steps) documentation.
+
+:::
+
+:::danger
+
+In Tuist integrated projects, there will be cases where `.xcworkspace` and `.xcodeproj` files will be created after the `tuist generate` command. For this reason, the **auto fill** feature in the **build configuration** may not work as expected. For more information abour build configurations, please visit the [**Build Configurations**](/build/build-process-management/build-profile-configuration) documentation.
+
+:::
+
 <Screenshot url='https://cdn.appcircle.io/docs/assets/BE2829-tuistOrder.png' />
 
 ### Input Variables
 
 You can find all the parameters required for this step in the table below, with their descriptions in detail.
 
+<Screenshot url='https://cdn.appcircle.io/docs/assets/BE4430New-tuistInput.png' />
 
-<Screenshot url='https://cdn.appcircle.io/docs/assets/BE2829-tuistInput.png' />
-
-| Variable Name                            | Description                         | Status           |
-|-------------------------------|------------------------------------------------|------------------|
-| `$AC_TUIST_PATH`              | Specifies the path to the directory containing the project definition. This path is automatically generated after the [Git Clone](/workflows/common-workflow-steps/git-clone) step.  | Optional |
-| `$AC_TUIST_PROJECT_ONLY`      | Generates only the local project, without generating its dependencies. | Optional |
+| Variable Name            | Description                                                                                                                                                                         | Status   |
+|--------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|----------|
+| `$AC_TUIST_PATH`         | Specifies the path to the directory containing the project definition. This path is automatically generated after the [Git Clone](/workflows/common-workflow-steps/git-clone) step. | Optional |
```

Comment:
I'm not sure, but if the `$AC_TUIST_PROJECT_ONLY` variable is not used, wouldn't the `$AC_TUIST_PATH` variable be required?

Reply:
Generate project only feature has been depricated. It did not work properly. We removed the parameter. On the other hand, tuist path is optional but its pre-defined in the code as repository direction. Because user can change the directory.

Reply:
I see 👍🏻

Reply:
@boztopuz `$AC_TUIST_PATH` is marked as "required" in the component input. So, it should also be synced here as "required".